packages/cli/esbuild.ts

```diff
@@ -11,6 +12,7 @@ const watch = process.argv.includes('--watch')
 const getPath = (file: string) => resolve(__dirname, file)
 
 build({
+	plugins: [nativeNodeModulesPlugin],
```

Comment:
Can you build all other packages? (run `npm run build` from the root directory)
Since `typesafe-i18n` uses multiple `esbuild` script, I'm assuming it will fail on others too.

Reply:
Yes, I can build the other packages. Only `cli` failed for me.